parl/algorithms/policy_gradient_algorithm.py

```diff
@@ -0,0 +1,40 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle.fluid as fluid
+from parl.framework.algorithm_base import Algorithm
+import parl.layers as layers
+
+
+class PolicyGradientAlgorithm(Algorithm):
```

Comment:
This class name is too long.
The class provided in our framework should be as short as possible. Only extending the name length on the condition that it might be ambiguous is such a short length.